packages/next/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@open-condo/next",
```

Comment:
Should it be built like icons / ui / bridge? Or it's more like folder with ts files for now?) If so, should it specified in transpileModules of apps?

Reply:
It was just a folder with files, everything remains as before :)

Reply:
Technically it was valid JS package, since it met all the requirements and does not require the transpilation:
1. package-name
2. deps
3. main for file imports

Now it's not a package anymore, since ts packages are requiring transpiling to generate cjs / esm, which other apps can consume. also there's no  main / export fields anymore, which make it more similar to folder.

Reply:
> Technically it was valid JS package

returned it to a state that satisfies these points

> Should it be built like icons / ui / bridge?

I don't think this needs to be built. We do not plan to publish this package. It will be the like `@open-condo/codegen`

Reply:
> I don't think this needs to be built. We do not plan to publish this package...

I don't think so. There's a task for generating next and ks apps from CLI. I want next app to support Next 13+, intl and Apollo-client (with cache / SSR / proper codegen) out of the box. So, the intl is there. But I'm not really sure we should maintain `@open-condo/next` package. Apollo can be installed with ~100 lines of code, and intl with ~20. And we don't need all this `WithAnotherWrapper` anymore...

So, what do you think?

Reply:
Maybe todo to rename it to legacy-next or smth?

Reply:
Docs about transpiling?

Reply:
> Maybe todo to rename it to legacy-next or smth?

Added todo

> Docs about transpiling?

Added docs